session/bootstrap.go

```diff
@@ -248,6 +248,16 @@ const (
 		DEFAULT_ROLE_USER char(32) COLLATE utf8_bin NOT NULL DEFAULT '',
 		PRIMARY KEY (HOST,USER,DEFAULT_ROLE_HOST,DEFAULT_ROLE_USER)
 	)`
+
+	// CreateStatsTopNStoreTable stores topn data of a cmsketch with top n.
+	CreateStatsTopNStoreTable = `CREATE TABLE if not exists mysql.stats_topnstore (
+		table_id bigint(64) NOT NULL,
```

Comment:
Do we need to store database id as well?

Reply:
no need to, table id is unique in the whole storage.

Reply:
@erjiaqing Could you add some comments about each field?